WebContent/js/components/JobInstance.js

```diff
@@ -95,14 +95,18 @@ class JobInstance extends React.Component {
         const statusStyleActive = { display: 'inline' };
         const statusStyleAbend = { color: 'red', display: 'inline' };
         const statusStyleComplete = { color: 'grey', display: 'inline' };
-        if (job.get('returnCode')) {
-            if (job.get('returnCode').toLowerCase().includes('abend') || job.get('returnCode').toLowerCase().includes('jcl error')) {
-                return (<div style={statusStyleAbend}> [{job.get('returnCode')}]</div>);
+        if (job.get('status').toLowerCase().includes('output')) {
+            if (job.get('returnCode')) {
+                if (job.get('returnCode').toLowerCase().includes('abend') || job.get('returnCode').toLowerCase().includes('jcl error')
+                    || job.get('returnCode').toLowerCase().includes('sys fail') || job.get('returnCode').toLowerCase().includes('conv error')) {
+                    return (<div style={statusStyleAbend}> [{job.get('returnCode')}]</div>);
+                }
+                if (job.get('returnCode').toLowerCase().includes('cc') || job.get('returnCode').toLowerCase().includes('canceled')) {
+                    return (<div style={statusStyleComplete}> [{job.get('returnCode')}]</div>);
+                }
+                return (<div style={statusStyleActive}> [{job.get('returnCode')}]</div>);
             }
-            if (job.get('returnCode').toLowerCase().includes('cc') || job.get('returnCode').toLowerCase().includes('canceled')) {
-                return (<div style={statusStyleComplete}> [{job.get('returnCode')}]</div>);
-            }
-            return (<div style={statusStyleActive}> [{job.get('returnCode')}]</div>);
+            return (<div style={statusStyleComplete}> [{job.get('status')}]</div>);
         } else if (job.get('status')) {
```

Comment:
We should remove the if condition here to avoid not being able to render a job, so just make it a generic else.

For example if for some reason z/OSMF is unable to provide us with a job status we would still want  the job to appear.

This will mean checking for status before adding it in the return, this can be done inline, something like: `return <div style={statusStyleActive}> [{job.get('status') ? job.get('status') : ''}]</div>;`